save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationVulnerabilitiesTab.kt

```diff
@@ -66,20 +65,32 @@ private val vulnerabilityTable: FC<TableProps<VulnerabilityDto>> = tableComponen
 
 @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
 val organizationVulnerabilitiesTab: FC<OrganizationVulnerabilitiesMenuProps> = FC { props ->
-    vulnerabilityTable {
-        getData = { _, _ ->
-            get(
-                url = "$apiUrl/vulnerabilities/by-organization-and-status",
-                params = jso<dynamic> {
-                    organizationName = props.organizationName
-                    status = VulnerabilityStatus.APPROVED
-                },
-                headers = jsonHeaders,
-                loadingHandler = ::noopLoadingHandler,
-                responseHandler = ::noopResponseHandler,
-            ).unsafeMap {
-                it.decodeFromJsonString()
+    val (vulnerabilities, setVulnerabilities) = useState<Array<VulnerabilityDto>>(emptyArray())
+    useRequest {
+        val fetchedVulnerabilities: Array<VulnerabilityDto> = get(
+            url = "$apiUrl/vulnerabilities/by-organization-and-status",
+            params = jso<dynamic> {
+                organizationName = props.organizationName
+                status = VulnerabilityStatus.APPROVED
+            },
+            headers = jsonHeaders,
+            loadingHandler = ::noopLoadingHandler,
+            responseHandler = ::noopResponseHandler,
+        ).unsafeMap {
+            it.decodeFromJsonString()
+        }
+        setVulnerabilities(fetchedVulnerabilities)
+    }
+    div {
+        className = ClassName("col-8 mx-auto mt-1 mb-3")
+        if (vulnerabilities.isNotEmpty()) {
+            vulnerabilityTable {
+                getData = { _, _ ->
+                    vulnerabilities
+                }
             }
+        } else {
+            renderTablePlaceholder("text-center p-4 bg-white") { +"No vulnerabilities were found for this organization." }
```

Comment:
may be we can add some button or links here too, in case of empty list of vulns

like `create new vuln` or `add from existing list of vulns`